ironfish/src/rpc/routes/accounts/getTransactions.ts

```diff
@@ -109,3 +103,74 @@ const streamTransaction = async (
 
   request.stream(serialized)
 }
+
+const handleSingleTransaction = async (
+  request: RpcRequest<GetAccountTransactionsRequest, GetAccountTransactionsResponse>,
+  node: IronfishNode,
+  account: Account,
+  hash: string,
+): Promise<void> => {
+  const hashBuffer = Buffer.from(hash, 'hex')
+
+  const transaction = await account.getTransactionByUnsignedHash(hashBuffer)
+
+  if (transaction) {
+    await streamTransaction(request, node, account, transaction)
+  }
+}
+
+const handleLimitedTransactions = async (
+  request: RpcRequest<GetAccountTransactionsRequest, GetAccountTransactionsResponse>,
+  node: IronfishNode,
+  account: Account,
+  limit: number,
+  headSequence?: number | null,
+): Promise<void> => {
+  const queue = new FastPriorityQueue<TransactionValue>(function (a, b) {
+    if (a.sequence && b.sequence) {
+      // both a and b are mined on chain, use sequence as sort key
+      return a.sequence < b.sequence
+    } else {
+      // at least one is in pending status, use expirationSequence as sort key
+      if (a.transaction.expirationSequence() && b.transaction.expirationSequence()) {
+        // no minersFee transaction, use expirationSequence as sort key directly
+        return a.transaction.expirationSequence() < b.transaction.expirationSequence()
+      } else {
+        // minersFee transaction without sequence is always latest
```

Comment:
```suggestion
        // transactions without expiration are always latest
```
nit -- transactions with expirationSequence === 0 aren't necessarily miners fees